scripts/steamdb/global.js

```diff
@@ -107,3 +107,59 @@ GetOption( { 'steamdb-highlight': true }, ( items ) =>
 		}
 	} );
 } );
+
+GetOption( { 'steamdb-highlight-family': true }, ( items ) =>
+{
+	if( !items[ 'steamdb-highlight-family' ] )
+	{
+		return;
+	}
+
+	SendMessageToBackgroundScript( {
+		contentScriptQuery: 'FetchSteamUserFamilyData',
+	}, ( response ) =>
+	{
+		const OnPageLoaded = () =>
+		{
+			if( response.error )
+			{
+				if( response.error === 'You are not part of any family group.' )
+				{
+					WriteLog( response.error );
+				}
+				else
+				{
+					WriteLog( 'Failed to load family userdata', response.error );
+
+					window.postMessage( {
```

Comment:
I kind of expect this fail to fail a lot lol, let's not trigger an alert for now